packages/build/src/plugins_core/edge_functions/index.js

```diff
@@ -69,6 +65,12 @@ const coreStep = async function ({
 
   await validateEdgeFunctionsManifest({ buildDir, constants: { EDGE_FUNCTIONS_DIST: distDirectory } })
 
+  if (!isRunningLocally) {
+    const logsLink = `https://app.netlify.com/sites/${env.SITE_NAME}/edge-handlers?scope=deployid:${env.DEPLOY_ID}`
+    const summaryText = `${declarations.length} Edge Functions deployed. [Watch Logs](${logsLink})`
```

Comment:
We should only use the capitalised version when referring to the product. When referring to actual instances of functions, we should use lowercase.

```suggestion
    const summaryText = `${declarations.length} edge functions deployed. [Watch Logs](${logsLink})`
```